mpp/shmemx_c_func.h4

```diff
@@ -191,6 +191,8 @@ define(`SHMEM_C_CTX_PUT_N_SIGNAL_NBI',
 SHMEM_DECLARE_FOR_SIZES(`SHMEM_C_CTX_PUT_N_SIGNAL_NBI')
 SHMEM_C_CTX_PUT_N_SIGNAL_NBI(mem,1);
 
+/* Signal fetch */
+SHMEM_FUNCTION_ATTRIBUTES uint64_t SHPRE()shmemx_signal_fetch(const uint64_t *sig_addr);
 
```

Comment:
@davidozog I see it in the original PR here. Not sure how it is not merged?

Reply:
@wrrobin Oh, I see it too - this might be my mistake working with a stale file or something...  I'll check.